snc.sh

```diff
@@ -86,7 +86,7 @@ if test -z "${SNC_OPENSHIFT_INSTALL_NO_DEBUG-}"; then
 else
         OPENSHIFT_INSTALL_EXTRA_ARGS=""
 fi
-
+BAREMETAL_RUNTIMECFG=$(${OC} adm release info -a ${OPENSHIFT_PULL_SECRET_PATH} ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE} --image-for=baremetal-runtimecfg)
```

Comment:
In the hunk just before this one, we switched away from `openshift-baremetal-install` to `openshift-install`, and here we set `BAREMETAL_RUNTIMECFG`? Maybe we should keep using the baremetal installer after all?

Reply:
This variable seems to be used once at the other end of the file, maybe we can keep both definitions together?

Reply:
> In the hunk just before this one, we switched away from openshift-baremetal-install to openshift-install, and here we set BAREMETAL_RUNTIMECFG? Maybe we should keep using the baremetal installer after all?

We used `openshift-baremetal-install` because it had libvirt provider option not because we were doing some baremetal installation.  So even we keep using `baremetal` installer we still need this image to be part of cluster.

> This variable seems to be used once at the other end of the file, maybe we can keep both definitions together?

Yes it is just to pull that image again after prune,  sure we can move it together.

Reply:
I know why we used `openshift-baremetal-install` before, but now we switch away from it, and we set a `BAREMETAL` variable, as if we were doing a baremetal install even if we stopped using the baremetal installer. Weird to see both changes at the same time :)